v3/docs/00-style-and-contributor-guidelines/a-contributor-guidelines/index.mdx

```diff
@@ -4,7 +4,7 @@ slug: /v3/contribute/style-guide
 version: '3.0'
 section: docs
 category: style guide
-keywords: contribute, style guide
+keywords: ['contribute', 'style guide']
```

Comment:
All of these need to be yml syntax.
```suggestion
keywords:
  - contribute
  - style guide
```

Reply:
example: https://assemble.io/docs/YAML-front-matter.html
https://yaml.org/

Reply:
Both is accepted, differece here is only that the array syntax is an one-liner.. the yaml format makes easier to write more complex data than just scalars but we don't need there. So really depends just on what is your writing preference in md files